backend/api/.env.dev

```diff
@@ -0,0 +1,9 @@
+DEBUG=1
```

Comment:
Typically .env files with authentication credentials are kept secret and not checked into source control. They should be excluded in .gitignore.

Reply:
I included production .env files in .gitignore, but checked in ```.env.dev```, since these values of the environment variables will only be used by developers building on their local machines. I suppose a more secure approach is to keep all ```.env``` files out of version control and set sane defaults for development in the python code, e.g. in ```os.environ.get```.

Reply:
@galbwe Oh, I may have misunderstood. I was thinking the .dev values could be used with development instances in the cloud, not just localhost. I don't want development instances to be vulnerable.

Reply:
@coloradocarlos It's probably best to err on the side of safety and leave .env files out of version control. It turns out I missed a few environment variables that are read into ```settings.py``` anyway, so I'm glad you brought it up.